wallet/src/de/schildbach/wallet/ui/transactions/TransactionResultViewBinder.kt

```diff
@@ -67,22 +72,25 @@ class TransactionResultViewBinder(private val containerView: View) {
     private val paymentMemoContainer by lazy { containerView.findViewById<View>(R.id.payment_memo_container) }
     private val payeeSecuredByContainer by lazy { containerView.findViewById<View>(R.id.payee_verified_by_container) }
     private val payeeSecuredBy by lazy { containerView.findViewById<TextView>(R.id.payee_secured_by) }
-    private val closeIcon by lazy { containerView.findViewById<ImageButton>(R.id.close_btn) }
```

Comment:
Close buttons are removed from the `transaction_result_content` since every host has its own and can handle it locally.